03/app.js

```diff
@@ -1,5 +1,34 @@
+/*Tym razem Twoim zadaniem jest napisanie kodu, który pozwoli po kliknięciu w <button/> pobrać number IP z jakim łączysz się z Internetem i wstawić go do elementu <span/>.
+
+Do rozwiązanie tego zadania możesz użyć API dostępnego pod tym adresem: https://www.ipify.org. Dane odbierane z API powinny być w formacie JSON.*/
+
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     console.log('DOM');
+
+    const button = document.querySelector('button');
+    button.addEventListener('click', getNumber);
+}
+
+const getNumber = () => {
+    const promise = fetch('https://www.ipify.org');
+
+    promise
+        .then(resp => {
+            if (resp.ok) { return resp.json(); }
+            return Promise.reject(resp);
+        })
+
+        .then(ip => {
+            console.log(ip);
+            const span = document.querySelector('.span');
```

Comment:
Nie ma elementu o tej klasie, chyba, że chcesz szukać po elemencie to wtedy samo `span`